app/graphql/mutations/app_packages/update_package.rb

```diff
@@ -17,7 +17,7 @@ def resolve(app_key:, id:, params:)
           app: app
         }
 
-        authorize! app, to: :manage?, with: AppPolicy
+        # authorize! app, to: :manage?, with: AppPolicy
```

Comment:
is it ok to comment this line?

Reply:
ah, right! then it is safe to remove it. 
note that there is a failing spec on cypress on the app_packages, maybe is due to this change